lib/widgets/comment/comment.dart

```diff
@@ -242,25 +246,29 @@ class _CommentWidget extends HookWidget {
                             ),
                           ),
                         ),
-                      Expanded(
-                        child: InkWell(
-                          onTap: () =>
-                              goToUser.fromPersonSafe(context, creator),
-                          child: Text(
-                            creator.originPreferredName,
-                            maxLines: 1,
-                            overflow: TextOverflow.ellipsis,
-                            style: TextStyle(
-                              fontSize:
-                                  context.read<ConfigStore>().commentTitleSize,
-                              color: theme.colorScheme.secondary,
-                            ),
+                      InkWell(
+                        onTap: () => goToUser.fromPersonSafe(context, creator),
+                        child: Text(
+                          creator.originPreferredName,
+                          maxLines: 1,
+                          overflow: TextOverflow.ellipsis,
+                          style: TextStyle(
+                            fontSize:
+                                context.read<ConfigStore>().commentTitleSize,
+                            color: theme.colorScheme.secondary,
                           ),
                         ),
                       ),
                       if (creator.isCakeDay) const Text(' 🍰'),
+                      if (creator.id ==
+                          (_tryGetUserData(context, store.comment.instanceHost)
+                                  ?.userId ??
+                              -1))
```

Comment:
Why is this `?? -1` needed? the `==` operator can compare with `null` just fine.

Reply:
Good point, thanks